pkg/plugin/plugin.go

```diff
@@ -240,3 +240,17 @@ func (p *plugin) GetControlPlanePods(ctx context.Context, oc *api.OpenShiftManag
 	}
 	return json.Marshal(pods)
 }
+
+func (p *plugin) BackupEtcdCluster(ctx context.Context, oc *api.OpenShiftManagedCluster, backupName string) (string, *api.PluginError) {
+	p.log.Info("generating admin kubeclient")
+	err := p.initialize(ctx, oc)
+	if err != nil {
+		return "", &api.PluginError{Err: err, Step: "InitializeAdminKubeClient"}
```

Comment:
must use predefined consts for PluginError Step so that we know what all the possible return values are in one place